fdbserver/fdbserver.actor.cpp

```diff
@@ -701,6 +702,7 @@ static void printUsage(const char* name, bool devhelp) {
 	printOptionUsage("--blob-credentials FILE",
 	                 "File containing blob credentials in JSON format. Can be specified "
 	                 "multiple times for multiple files. See fdbbackup usage for more details.");
+	printOptionUsage("--proxy MACHINE", "IP:port or host:port to proxy server for connecting network.");
```

Comment:
This is probably not accurate. We want to say this this proxy for "connecting to backup's blob store". I.e., `fdbserver` communications do not go through this proxy.

Reply:
Changed. Thanks!